internal/connector/internal/config/connectors.go

```diff
@@ -119,3 +134,43 @@ func checksum(spec interface{}) (string, error) {
 	}
 	return fmt.Sprintf("%x", h.Sum(nil)), nil
 }
+
+func walk(filename string, linkDirname string, walkFn filepath.WalkFunc) error {
```

Comment:
It seems quite generic and useful function, should we move it to `pkg/shared/utils` ?

Reply:
Will do

Reply:
this pr

Reply:
done